xbmc/GUIInfoManager.cpp

```diff
@@ -5017,6 +5023,20 @@ std::string CGUIInfoManager::GetItemLabel(const CFileItem *item, int info, std::
     return StringUtils::Format("%i", item->GetVideoInfoTag()->GetRating(votes).votes);
   }
 
+  if (info >= LISTITEM_PROPERTY_START + LISTITEM_RATINGANDVOTES_OFFSET && info - (LISTITEM_PROPERTY_START + LISTITEM_RATINGANDVOTES_OFFSET) < (int)m_listitemProperties.size())
+  { // grab the rating and the votes
+    std::string ratingName = m_listitemProperties[info - (LISTITEM_PROPERTY_START + LISTITEM_RATINGANDVOTES_OFFSET)];
+    CRating rating = item->GetVideoInfoTag()->GetRating(ratingName);
+
+    if (rating.rating <= 0.f)
+      return "";
+    
+    if (rating.votes == 0)
+      return StringUtils::Format("%.1f", rating.rating);
+    else
+      return StringUtils::Format("%.1f (%i %s)", rating.rating, rating.votes, g_localizeStrings.Get(20350).c_str());
```

Comment:
You should move `%.1f (%i votes)` (or at least the `%i votes` part) to the language file so it can be properly translated.